driver/src/orderbook/streamed/bigint_u256.rs

```diff
@@ -0,0 +1,57 @@
+use ethcontract::U256;
+use num::{bigint::Sign, BigInt, BigUint};
+
+/// None if U256 cannot represent the number.
+pub fn bigint_to_u256(n: &BigInt) -> Option<U256> {
+    match n.to_bytes_le() {
+        (Sign::NoSign, _) => Some(U256::zero()),
+        (Sign::Plus, bytes) if bytes.len() <= 256 / 8 => Some(U256::from_little_endian(&bytes)),
+        _ => None,
+    }
+}
+
+pub fn u256_to_biguint(n: U256) -> BigUint {
+    let mut bytes = [0u8; 256 / 8];
+    n.to_little_endian(&mut bytes);
+    BigUint::from_bytes_le(&bytes)
+}
+
+pub fn u256_to_bigint(n: U256) -> BigInt {
+    BigInt::from_biguint(Sign::Plus, u256_to_biguint(n))
+}
```

Comment:
Could any of these be a `From` or `TryFrom` implementation? Is there some reason you have decided against this?

Reply:
It's not possible because in Rust either the trait or the type on which we implement the trait has to come from this module. In this case U256 and BigInt come from other crates and From comes from the standard library.